cfpb/consumerfinance.gov
# Rebuild Elasticsearch 7 indexes on each deploy

Now that we have a flagged search route as well as a document to index, it seems prudent to begin indexing our Elasticsearch 7 containers on each deploy to our docker stack.

---




## Additions

- A force rebuild of all indexes within our codebase configured for Elasticsearch 7 on each deployment. (I know deploy time can be a concern, but the indexing for ES 7 seems to be much quicker than ES 2).


## How to test this PR

1. Deploy a new stack
2. Enable `ELASTICSEARCH_DSL_REGULATIONS` flag in the admin section for the stack
3. Query Regulations and confirm results are present.


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
